### PR TITLE
Update 20240508

### PR DESCRIPTION
**Pull request by eContento**
Move the current talk to previous and added the next talks about «Java cumple 29» and «TDD. En serio, ¡pruébalo!»

### CODE REVIEW

File: _pages/index.html [hunk 1]
```diff
@@ -40,21 +40,32 @@
...
+      image-width: "500px"
+      image-right: true
+    - header: "TDD. En serio, ¡pruébalo!"
+      date: 23 Mayo 2024
+      #excerpt: "En esta charla pretendo contaros todos los aprendizajes que he tenido colaborando con el proyecto https://justicio.es/. Tratar de separar un el polvo de la paja, y hablaros de ciertos retos de ingeniería que no se suelen comentar. Hablaremos del rol que tiene clave el hardware, las bases de datos de vectores, la importancia y descuidada tarea de la tokenización, y propondré algún dilema que otro sobre la industria ;-)"
+      #btn-url: "https://www.meetup.com/madridjug/events/300631292/"
+      #btn-class: "info"
+      #btn-label: "Registro"
+      blank: true
+      image-url: "assets/images/20240523-TDD.png"
```
Comment (ilopmar) [inline _pages/index.html:61]: The `except` is duplicated and it's from the talk we have today, not the next one

File: _pages/index.html [hunk 2]
```diff
@@ -40,21 +40,32 @@
...
+      #btn-url: "https://www.meetup.com/madridjug/events/300631292/"
+      #btn-class: "info"
+      #btn-label: "Registro"
       blank: true
-      image-url: "assets/images/20240429-TomateLaPasiLLIA.png"
+      image-url: "assets/images/20240523-JavaCumple29.png"
+      alt-text: "Cartel anunciando el evento"
+      image-width: "500px"
+      image-right: true
+    - header: "TDD. En serio, ¡pruébalo!"
```
Comment (aureamunoz) [inline _pages/index.html:54]: The header is wrong, it's from the previous event

**Review by aureamunoz — CHANGES_REQUESTED** [review]

File: _pages/index.html (hunk 2 above)
Comment (eContento) [reply _pages/index.html:54]: The next sessions are «Java cumple 29» and «TDD. En serio, ¡pruébalo». 
The previous event will be «Tómate la PastiLLIA Ante la fiebre de la IA y los LLM’s»

**Comment by ilopmar** [comment]
Just to confirm because I'm not sure, we are going to have 2 meetups announced, right? Even though they are the same day, but just to have the birthday and Nacho's talk.

File: _pages/index.html [hunk 3]
```diff
@@ -40,21 +40,34 @@
...
 comming-up-row-one:
-    - header: "Tómate la PastiLLIA Ante la fiebre de la IA y los LLM’s"
-      date: 8 Mayo 2024
-      excerpt: "En esta charla pretendo contaros todos los aprendizajes que he tenido colaborando con el proyecto https://justicio.es/. Tratar de separar un el polvo de la paja, y hablaros de ciertos retos de ingeniería que no se suelen comentar. Hablaremos del rol que tiene clave el hardware, las bases de datos de vectores, la importancia y descuidada tarea de la tokenización, y propondré algún dilema que otro sobre la industria ;-)"
-      btn-url: "https://www.meetup.com/madridjug/events/300631292/"
-      btn-class: "info"
-      btn-label: "Registro"
+    - header: "Java cumple 29"
+      date: 23 Mayo 2024
+      excerpt: "Nuestra querida plataforma Java va cumpliendo años, creciendo y haciendose cada vez más robusta. El próximo 23 de Mayo nos juntaremos para celebrar su 29 cumpleaños."
```
Comment (aureamunoz) [inline _pages/index.html:45]:
```suggestion
      excerpt: "¡Únete a nosotros para celebrar el 29 aniversario Java en Madrid! Desde su creación hace casi tres décadas, Java ha sido un pilar fundamental en el mundo del desarrollo de software, impulsando la innovación y la creación de aplicaciones que han transformado nuestra forma de interactuar con la tecnología. En este evento especial, conmemoraremos el éxito de Java, tendremos a un ponente de excepción compartiendo su sabiduría. ¡No te pierdas esta oportunidad de conectar con la comunidad de usuarios y celebrar juntos este hito histórico!"
```

**Review by aureamunoz — CHANGES_REQUESTED** [review]

**Pull request merged**